src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -428,7 +427,16 @@ bool Terminal::SendKeyEvent(const WORD vkey, const WORD scanCode, const ControlK
         return false;
     }
 
-    const auto ch = _CharacterFromKeyEvent(vkey, scanCode, states);
+    // By default Windows treats Ctrl+Alt as an alias for AltGr.
+    // When the altGrAliasing setting is set to false, this behaviour should be disabled.
+    //
+    // Whenever possible _CharacterFromKeyEvent() will return a valid character.
+    // For instance both Ctrl+Alt+Q as well as AltGr+Q return @ on a German keyboard.
+    //
+    // We can achieve the altGrAliasing functionality by skipping the call to _CharacterFromKeyEvent,
+    // as TerminalInput::HandleKey will then fall back to using the vkey which
+    // is the underlying ASCII character (e.g. A-Z) on the keyboard in our case.

```

Comment:
minor: I'd maybe include a comment like the following:
```suggestion
    // is the underlying ASCII character (e.g. A-Z) on the keyboard in our case.
    // See GH#5525/GH#6211 for more details
```